web/src/main/webapp/js/instructions.js

```diff
@@ -91,6 +89,27 @@ module.exports.create = function (mapLayer, path, urlForHistory, request) {
         instructionsElement.append(moreDiv);
     }
 
+    if (debugDetails) {
+        var detailObj = path.details;
+        for (var detailKey in detailObj) {
+            var pathDetailObj = detailObj[detailKey]["details"];
+            for (var k in pathDetailObj) {
+                var intervalArr = pathDetailObj[k];
+                for (var intervalIndex in intervalArr) {
+                    var interval = intervalArr[intervalIndex];
+                    var lngLat = path.points.coordinates[interval[0]];
+                    L.marker([lngLat[1], lngLat[0]], {
+                        icon: L.icon({
+                            iconUrl: './img/marker-small-blue.png',
+                            iconSize: [15, 15]
+                        }),
+                        draggable: true
+                    }).addTo(mapLayer.getRoutingLayer()).bindPopup(detailKey+":"+k);
+                }
+            }
+        }
+    }
```

Comment:
Hmmh, we either can have a compact way for path details which won't be a GeoJSON (current state) but probably it would be nice to have something which is easy to display directly as a GeoJSON. Similar to what is possible for the route geometry with points_encoded. But probably also for a different PR